.github/workflows/ci.yaml

```diff
@@ -221,6 +221,11 @@ jobs:
       - name: Add SBT proxy repositories
         run: cp -vf .github/workflows/repositories /root/.sbt/ ; true
 
+      - name: Disable experimental features on release
+        if: "github.event_name == 'push'
+            && startsWith(github.event.ref, 'refs/tags/')"
+        run: echo "DISALLOW_EXPERIMENTAL_FEATURES=true" >> $GITHUB_ENV
```

Comment:
Why not set `RELEASEBUILD` to `yes` like in publish_release here? That way we're actually testing with the version number we'll be using for publishing, and the existing compilerSupportExperimental flag should work.

Reply:
`dottyVersion` in `Build.scala` is set based on `RELEASEBUILD`. If it is a `yes`, `publishLocal` will publish the compiler locally without the `-bin-SNAPSHOT` prefix, thus breaking the community build. Hence the need for a separate env variable.

Reply:
> publishLocal will publish the compiler locally without the -bin-SNAPSHOT prefix, thus breaking the community build.

Why does this break the community build?